raiden/lightclient/models/light_client_payment.py

```diff
@@ -34,7 +34,7 @@ def __init__(
         self.partner_address = to_checksum_address(partner_address)
         self.is_lc_initiator = is_lc_initiator
         self.token_network_id = to_checksum_address(token_network_id)
-        self.amount = amount
+        self.amount = int(amount)
```

Comment:
Why is the amount changed to be received as `string` if it will be casted to `int ` and it was already received as an `int`?

Reply:
You're right, as we talk in the daily, i'll change this for a better operation in the code.